Formula/okta-aws-cli.rb

```diff
@@ -0,0 +1,22 @@
+class OktaAwsCli < Formula
+  desc "Okta authentication in support of aws cli operation"
```

Comment:
Taken from help text:

```suggestion
  desc "Okta federated identity for AWS CLI"
```